plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/BaseMongoConnectorTest.java

```diff
@@ -136,6 +135,17 @@ public void testGuessFieldTypes(String mongoValue, String trinoValue)
         };
     }
 
+    @Test
+    public void testCreateTableWithColumnComment()
+    {
+        try (TestTable table = new TestTable(getQueryRunner()::execute, "test_column_comment", "(col integer COMMENT 'test')")) {
```

Comment:
I'd suggest removing the TODO in BaseConnectorTest `// TODO: comment set when creating a table` and moving those cases to new test instead of adding connector specific test here.

Reply:
I won't touch in this PR since it's handled in #11162

Reply:
Can you add a TODO to move to the common test once https://github.com/trinodb/trino/issues/11162 is done. It'll make us not try to find all coulmn comment tests.